src/cbc_sdk/rest_api.py

```diff
@@ -250,7 +273,7 @@ def bulk_threat_dismiss(self, threat_ids, remediation=None, comment=None):
         """
         return self._bulk_threat_update_status(threat_ids, "DISMISSED", remediation, comment)
 
-    # ---- ThreathHunter
+    # ---- ThreatHunter
```

Comment:
This should be Enterprise EDR

Reply:
Heh, I fixed the typo, which is the only reason this showed up. :) Perhaps I should add another ticket to do a sweep of the comments for any outdated terminology?